system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.hpp

```diff
@@ -109,8 +107,17 @@ class PeriodicMeasurementNode : public system_metrics_collector::Collector, publ
    * If true, then measurements are cleared after publishing data
    */
   const bool clear_measurements_on_publish_;
+
   rclcpp::TimerBase::SharedPtr measurement_timer_;
   rclcpp::TimerBase::SharedPtr publish_timer_;
+
+protected:
+  /**
+   * Track the starting time of the statistics
+   */
+  rclcpp::Time window_start_;
+
+  rclcpp::Publisher<metrics_statistics_msgs::msg::MetricsMessage>::SharedPtr publisher_;
```

Comment:
The order for classes is public -> protected -> private (not public -> private -> protected like here).
See https://google.github.io/styleguide/cppguide.html#Declaration_Order

> A class definition should usually start with a public: section, followed by protected:, then private:. Omit sections that would be empty.

Reply:
Done.